### DISCUSSION TITLE
Include sorting capability

### DISCUSSION THREAD

**Discussion by vadimkholodilo** [bitwarden/clients#6074] [Password Manager]
### Code Contribution Proposal

I would like to implement an ability to sort items by various fields such as modified date. This feature is requested by a lot of users. See this forum topic: https://community.bitwarden.com/t/sort-items-by-date-of-modification-addition-last-use-etc/2484/5
However, I'm not sure where the sorting should be implemented: on the backend or front-end. If there are not many entries, front-end would be the best place.

**Comment by dwbit** [comment]
Hey @vadimkholodilo is this a feature request, or code that you would like to contribute to the Bitwarden codebase?

**Comment by vadimkholodilo** [comment]
I can write code for this feature

**Comment by dwbit** [comment]
Thanks for confirmation! The team will review and get back to you shortly.

**Comment by livgm** [comment]
@dwbit Just wondering what the current state of this is - if @vadimkholodilo is not interested in working on this anymore, I am happy to cover this as well!

**Comment by vadimkholodilo** [comment]
Unfortunately, I don’t have time right now to work on this issue.

**Comment by dwbit** [comment]
Closing this older discussion for now, feel free to open a new discussion if needed.
